pkg/suggestion/v1beta1/skopt/base_service.py

```diff
@@ -118,13 +118,13 @@ def getSuggestions(self, trials, request_number):
 
         return_trial_list = []
 
-        skopt_suggested = self.skopt_optimizer.ask(n_points=request_number)
+        skopt_suggested = self.skopt_optimizer.ask(n_points=current_request_number)
         for suggestion in skopt_suggested:
             logger.info("New suggested parameters for Trial: {}".format(suggestion))
             return_trial_list.append(
                 BaseSkoptService.convert(self.search_space, suggestion))
 
-        logger.info("GetSuggestions return {} new Trials\n\n".format(request_number))
+        logger.info("GetSuggestions returns {} new Trials\n\n".format(current_request_number))
```

Comment:
maybe it's not about this PR directly, but isn't this log should be `len(skopt_suggested)` or `len(return_trial_list)` rather than `current_request_number` ?

Reply:
Nice catch!